pages/about/groups/task-forces/pdf-aam/work-statement.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Portable Document Format Accessibility APIs Mapping Task Force Work Statement
+permalink: /about/groups/task-forces/pdf-aam/work-statement/
+ref: /about/groups/task-forces/pdf-aam/work-statement/
+lang: en
+github:
+  label: wai-groups
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Summary" class="" %}
+{:/}
+
+The Portable Document Format Accessibility <abbr title="Application Programming Interfaces">APIs</abbr> Mapping (PDF-AAM) Task Force is a task force of the [Accessible Rich Internet Applications (ARIA) Working Group](/about/groups/ariawg/). It assists this Working Group with the work identified below.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2..3" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Status
+
+The PDF-AAM Task Force Work Statement was [approved on [dd mm yyyy]](CFC_or_minutes_pointer) and is now active. The [PDF-AAM Task Force home page](/about/groups/task-forces/pdf-aam/) contains information about the operation and resources of the group.
+
+## Objective
+
+The mission of the PDF-AAM Task Force is to specify how user agents should map accessibility semantics within the Portable Document Format specification. This Task Force provides a focused forum for this work while the Working Group continues its work on ARIA development.
+
+## Scope of Work
+
+The PDF-AAM Task Force will produce a W3C Technical Report on [PDF Accessibility API Mappings 1.0](https://w3c.github.io/pdf-aam/), specifying accessibility API mappings for PDF, based on the [Core-AAM](https://www.w3.org/TR/core-aam-1.2/) and [HTML-AAM](https://www.w3.org/TR/html-aam/) specifications.
+
+## Approach
+
+The [PDF/UA Processor Liaison Working Group (LWG)](https://pdfa.org/community/pdf-ua-processor-lwg/) is a meeting place for the PDF and assistive technology industries to help drive the formalization of common understandings on processing PDF files that conform to [ISO 14289 (PDF/UA)](https://www.iso.org/standard/54564.html), the standard for accessible PDF documents. The PDF-AAM will incorporate and extend existing work developed by the PDF/UA Processor LWG into a W3C Technical Report, which is expected to reach the Recommendation Track according to the [W3C Process](https://www.w3.org/policies/process/).
```

Comment:
@daniel-montalvo - please check the link for the PDF standard. I think it should be [ISO 14289 (PDF/UA)](https://www.iso.org/standard/64599.html), but I'm not sure.